lora-tutorials/lora-notecard-application/lora-notecard-application.ino

```diff
@@ -0,0 +1,241 @@
+#include <Notecard.h>
+
+// Notefile Definitions
+#define	DATA_NOTEFILE		     "data.qo"
+#define	DATA_PORT            1
+#define INBOUND_NOTEFILE     "poll.qi"
+#define INBOUND_PORT         2
+
+#define FIELD_TEMPERATURE    "temperature"
+#define FIELD_VOLTAGE        "voltage"
+#define FIELD_POLL           "poll"
+
+#define INBOUND_NOTEFILE_POLL_SECS 10
+#define ENV_POLL_SECS              10
+#define DATA_SEND_MINS             5
+
+// Environment variable definitions
+#define	VAR_MINUTES	"minutes"
```

Comment:
I would call this `TVAR_MINUTES`, so you know it was templated.

Also, I don't really see how it's different from `FIELD_xxxx`, which I would also replace with `TVAR_xxxx`. They are all variables from a template, regardless of whether it's qi, qo or env.